schedview/dayobs.py

```diff
@@ -171,8 +165,7 @@ def from_time(
         if dayobs_datetime.tzinfo is None:
             dayobs_datetime = dayobs_datetime.replace(tzinfo=datetime.timezone.utc)
 
-        dayobs_datetime = dayobs_datetime.astimezone(DAYOBS_TZ)
-        dayobs_date = datetime.date(dayobs_datetime.year, dayobs_datetime.month, dayobs_datetime.day)
+        dayobs_date = dayobs_datetime.astimezone(DAYOBS_TZ).date()
```

Comment:
What is DAYOBS_TZ ? 
I'm just curious because dayobs is essentially defined in UTC and I suspect we don't want to mess with that too much ... otherwise people in vastly different timezones like Australia might end up locating visits on completely different days (or at least having confusing "day" labels).

Reply:
This is me being punctilious about following SITCOMTN-032 exactly, which states that dayObs is:

> the observation day (in timezone UTC-12:00) when the takeImages command began executing. 

which line 10 of `dayobs.py`:

`DAYOBS_TZ = datetime.timezone(datetime.timedelta(hours=-12))`

Reply:
I see, so it's not that DAYOBS_TZ can change, it's also defined.